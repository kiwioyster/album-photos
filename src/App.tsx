import React from 'react';
import './App.css';
import AlbumList from './components/AlbumList';

function App() {
  return (
    <div className='App'>
      <AlbumList></AlbumList>
    </div>
  );
}

export default App;
