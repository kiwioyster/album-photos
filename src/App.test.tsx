import React from 'react';
import { shallow, configure } from 'enzyme';
import App from './App';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
describe('App', () => {
  it('should render', () => {
    const component = shallow(<App />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
