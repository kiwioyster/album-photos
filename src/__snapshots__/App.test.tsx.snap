// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
Array [
  <div
    className="App"
  >
    <AlbumList />
  </div>,
]
`;
